testing/mocks/generic.go

```diff
@@ -435,6 +435,29 @@ func GenericAmount(delta int) cadence.Value {
 	return cadence.NewUInt64(random.Uint64())
 }
 
+func GenericSeals(number int) []*flow.Seal {
+	var seals []*flow.Seal
+	for i := 0; i < number; i++ {
+
+		seal := flow.Seal{
+			BlockID:    GenericIdentifier(2 * i),
+			ResultID:   GenericIdentifier(2*i + 1),
```

Comment:
I encountered a similar issue in this file (see `GenericCollections`) and did that indeed:

```go
	for i := 0; i < number; i++ {
		// Since we want two transactions per collection, we use a secondary index called `j` for the transaction IDs.
		j := i * 2
		collections = append(collections, &flow.LightCollection{Transactions: txIDs[j : j+1]})
	}
```

I agree that it's more readable :)